config/samples/subscription_oo.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  generation: 1
+  labels:
+    operators.coreos.com/cluster-observability-operator.openshift-operators: ""
+  name: cluster-observability-operator
+  namespace: openshift-operators
+spec:
+  channel: development
+  installPlanApproval: Automatic
+  name: cluster-observability-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: cluster-observability-operator.0.4.1
```

Comment:
hmm...I wonder is there a better way of configuring this? 
At some point 0.4.1 might be unavailable, I wonder what happens when you create sub without the startingCSV? would it install latest?